src/styles/services-view.less

```diff
@@ -7,6 +7,9 @@ services-view,
     padding-left: 2px;
     vertical-align: middle;
   }
+  pf-filter-panel .filter-panel-container ul {
```

Comment:
This is a bit of overkill because of where the rule is nested and the specificity.   Try

```
.filter-panel-container ul {
  margin-bottom: 10px;
}
```
on line 20.  Should produce the same result, but without duplication and overspecificity.

Reply:
Thanks, Fixed